scripts/depsvalidator/Package.swift

```diff
@@ -1,9 +1,9 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.5
 import PackageDescription
 
 let package = Package(
     name: "DepsValidator",
-    platforms: [.macOS(.v10_13)],
+    platforms: [.macOS(.v10_15)],
```

Comment:
Upgraded this because I was trying out a dependency that required it. I removed the dependency, but left this change since it seems unnecessary to be so aggressive with supporting old macOS versions.